src/prefect/run_configs/vertex.py

```diff
@@ -0,0 +1,77 @@
+from typing import Union, Iterable
+
+import yaml
+
+from prefect.run_configs.base import RunConfig
+from prefect.utilities.filesystems import parse_path
+
+
+class VertexRun(RunConfig):
+    """Configure a flow-run to run as an ECS Task.
+
+    Vertex Tasks are composed of and runtime parameters.
+
+    Runtime parameters can be specified via `run_task_kwargs`. These will be
+    merged with any runtime parameters configured on the agent when starting
+    the task.
+
+    Args:
+        - service_account (str):
+            Specifies the service account for workload run-as account.
+            Users submitting jobs must have act-as permission on this
+            run-as account. If unspecified, the `AI Platform Custom Code
+            Service
+            Agent <https://cloud.google.com/vertex-ai/docs/general/access-control#service-agents>`__
+            for the CustomJob's project is used.
+        - machine_type (str):
+            Needs to be one of the allowed machine types
+            https://cloud.google.com/vertex-ai/docs/training/configure-compute#machine-types
+        - boot_disk_size_gb (int)
+            Size in GB of the boot disk (default is 100GB).
+        - image (str, optional): The image to use for this task. If not
+            provided, will be either inferred from the flow's storage (if using
+            `Docker` storage), or use the default configured on the agent.
+        - env (dict, optional): Additional environment variables to set on the task.
+        - run_task_kwargs (dict, optional): Additional keyword arguments to
```

Comment:
nit: as far as I can tell this isn't actually supported anywhere (it's not saved on this config). Anyone know immediately what function this serves and if we need it?

Reply:
Yeah, some of these are legacy based on the ECS config. The run config was the most partially baked file.